### ISSUE TITLE
Double clicking required to activate buttons and checkboxes

### ISSUE THREAD

**Issue by michalfita** [OpenCppCoverage/OpenCppCoveragePlugin#28]
To activate **Toggle projects selection** button or any checkbox above, I have to double click every widget. First grabs focus, second allows to invoke the action. Same applies to filter rows in next tab, first click grabs focus, second enable edition. This is annoying.

Is there some weird focus grabbing layer implemented for this dialogue window?

I'm using Visual Studio 2017 15.9.9, but that happened in a few previous versions of 2017 either. I'm using the OpenCppCoveragePlugin only recently.

**Comment by OpenCppCoverage** [comment]
Hello,

Sorry for my late anwser.
Indeed, when you click on a cell, you enter in edit mode and then you can update the value. Do you know a better way to do it in WPF?

OpenCppCoverage

**Comment by michalfita** [comment]
I don't do WPF, so I don't know. From UX perspective is very unnatural, most forms I'm using require just one click to select the field as I don't need to click them for anything else than edition.

**Comment by OpenCppCoverage** [comment]
Indeed, it is not very natural. I think it is less annoying now, as settings are saved, but I will try to see if I can do something about it.

**Comment by tomv564** [comment]
The DataGrid used for the project list is suitable for an Excel-like editing experience of data.

A much simpler control like ListBox might be more appropriate.
I did a quick example at https://github.com/OpenCppCoverage/OpenCppCoveragePlugin/compare/master...tomv564:simpler-project-selection-box

Let me know if you'd like help with this!

**Comment by OpenCppCoverage** [comment]
Thank you for reporting this issue and for contributing to the project.

The project is now being formally retired and the repository will be archived. I am therefore closing all remaining issues, as no further investigation, fixes, or releases are planned.

This administrative closure does not mean that the issue was resolved or that the report was invalid. The discussion will remain available for anyone who wants to address it in a fork.

Thank you for your understanding.
